### DISCUSSION TITLE
infuse通过webdav连接alist挂载的阿里云盘，播放视频卡顿或者一直转圈怎么解决

### DISCUSSION THREAD

**Discussion by KJFKA** [AlistGo/alist#2981] [General]
哪位大神能告诉我怎么解决这情况，体验太差了，急急急！

**Comment by NovaShen0724** [comment]
试试302->本地代理

**Reply by KJFKA** [reply]
web代理需要开启吗？选302重向还是本地代理？

**Reply by Dashan-IZ** [reply]
我也想知道

**Comment by aomsir** [comment]
同样的问题，试了下302重定向，web代理开了以后重启了Alist好了

**Comment by 1273257194** [comment]
修改为本地代理成功了

**Comment by sjjsjss** [comment]
有没有本地代理 打开web代理 也卡的...前几天都好好的，昨晚开始打开视频一直卡顿

**Comment by xianwx** [comment]
从昨晚开始的，我看贴吧也这样，应该是阿里云又改了啥

**Reply by mm2175** [reply]
几天前好好的，今天看龟速

**Reply by GitCourser** [reply]
我也是昨天变慢,前天还是好的,肯定是阿里云开始限制以前的方法了,换成open的就好了

**Reply by sjjsjss** [reply]
具体怎么操作 大佬> 我也是昨天变慢,前天还是好的,肯定是阿里云开始限制以前的方法了,换成open的就好了

**Comment by MarsHou** [comment]
更新alist到最新V3.12.2，Storage 添加Aliyundrive Open 就OK了！

**Reply by sjjsjss** [reply]
我这v3.12.1就是最新的了，merlin alist

**Reply by MarsHou** [reply]
merlin 不是很清楚，你看看你新挂载一个网盘，选项里是不是有个Aliyundrive Open！

**Reply by sjjsjss** [reply]
看见了 谢谢！

**Reply by sjjsjss** [reply]
还有个问题 我已经添加了但是显示我只有阿里云的token没有open的咋办

**Reply by MarsHou** [reply]
没明白什么意思，你原来挂载的aliyundrive都可以删了（之前的方式弃用了，具体内容你可以看看alist的文档），重新挂载，token使用alist提供的授权链接扫码生成token

**Reply by sjjsjss** [reply]
果然有教程文档！已经搞好了，谢谢！大佬你是最棒的！> 没明白什么意思，你原来挂载的aliyundrive都可以删了（之前的方式弃用了，具体内容你可以看看alist的文档），重新挂载，token使用alist提供的授权链接扫码生成token

**Reply by zhoutao-cn** [reply]
可是，改用 OPEN 之后只解决了用户本人空间的连接问题；挂载他人的分享资源速度非常慢。
有没有解决方案？
